### PR TITLE
Add version to profile.md header

### PR DESCRIPTION
**Pull request by mattmcneeney**
Make it clear that this file is versioned

### CODE REVIEW

File: profile.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Open Service Broker API Profile
+# Open Service Broker API Profile v2.12
```
Comment (duglin) [inline profile.md]: hmm, should this say "(master - might contain changes that are not yet released)" to align with master?  If so, we should probably have a PR for the v2.12 branch that says "v2.12".

Comment (mattmcneeney) [inline profile.md]: @duglin That's exactly what I've done! 😄

Comment (duglin) [inline profile.md]: ah, I missed the branch name - sorry

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by fmui** [comment]
LGTM

**Comment by avade** [comment]
lgtm

**Pull request merged**